terminus/geometry/arc.py

```diff
@@ -221,6 +295,12 @@ def _angular_offset_for_point(self, point):
     def is_valid_path_connection(self):
         return self.radius() > 4.0
 
+    def almost_equal_to(self, other):
```

Comment:
If you are going to make this public, allow for an optional precision parameter (see `Point` implementation)